gcp-observability/src/test/java/io/grpc/gcp/observability/TracesTest.java

```diff
@@ -100,7 +100,7 @@ public void run() {
 
       when(mockConfig.isEnableCloudTracing()).thenReturn(true);
       when(mockConfig.getSampler()).thenReturn(Samplers.alwaysSample());
-      when(mockConfig.getDestinationProjectId()).thenReturn(PROJECT_ID);
+      when(mockConfig.getProjectId()).thenReturn(PROJECT_ID);
```

Comment:
Comment for line 121 (github doesn't let me add a comment there): `LoggingTestHelper` will need to be renamed since it is used for Metrics and Traces tests as well. May be call it `ObservabilityTestHelper`?

Reply:
Done.